src/main/scala/nl.knaw.dans.easy.license/Command.scala

```diff
@@ -67,6 +67,7 @@ object Command extends App with DebugEnhancedLogging {
           .doOnCompleted {
             logger.info(s"license saved at ${ outputFile.getAbsolutePath }")
             params.fsrdb.close()
+            params.ldap.close()
```

Comment:
since you're adding `params.ldap.close()` multiple times and `params.fsrdb.close()` is repeated as well, would it make sense to implement `AutoCloseable` on `params` (concretely on `DatabaseParameters`) and then call `params.close()`?

Reply:
A good idea. Done.